### ISSUE TITLE
Crashes when exporting an model with no animation upon removing useless joints.

### ISSUE THREAD

**Issue by SausageTaste** [SausageTaste/io_scene_dalbaragi#1]
If a model has a armature but no animations, the exporter will fail to optimize out useless joints.
Saying something like this:

```py
Traceback (most recent call last):
  File "C:\Users\woos8\AppData\Roaming\Blender Foundation\Blender\2.81\scripts\addons\io_scene_dalbaragi\__init__.py", line 529, in execute
    model = ModelBuilder(self.optionBool_removeUselessJoints)
  File "C:\Users\woos8\AppData\Roaming\Blender Foundation\Blender\2.81\scripts\addons\io_scene_dalbaragi\__init__.py", line 359, in __init__
    jointIndexMap = self.__skeleton.removeJoints(uselesses)
  File "C:\Users\woos8\AppData\Roaming\Blender Foundation\Blender\2.81\scripts\addons\io_scene_dalbaragi\datastruct.py", line 392, in removeJoints
    if joint.m_name in jointNames:
TypeError: argument of type 'NoneType' is not iterable
```

**Comment by SausageTaste** [comment]
The issue was partialy addressed in #8. But I'm planning to move the function to C++ tool so it shouldn't matter anymore.

**Comment by SausageTaste** [comment]
The error producing code has been deprecated so I shouldn't bother with this issue.
